src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -24,8 +24,11 @@ internal interface IFastTreeTrainerFactory : IComponentFactory<ITrainer>
     public sealed partial class FastTreeBinaryClassificationTrainer
     {
         [TlcModule.Component(Name = LoadNameValue, FriendlyName = UserNameValue, Desc = Summary)]
-        public sealed class Arguments : BoostedTreeArgs, IFastTreeTrainerFactory
+        public sealed class Options : BoostedTreeArgs, IFastTreeTrainerFactory
         {
+            /// <summary>
+            /// Should we use derivatives optimized for unbalanced sets?
+            /// </summary>
```

Comment:
Maybe for a summary description of an argument, it would be better to have a declarative sentence instead of a question. Could you change this to a declarative sentence?

Reply:
fixed

---
In reply to: [245963403](https://github.com/dotnet/machinelearning/pull/2047#discussion_r245963403) [](ancestors = 245963403)